cmd/restic/cmd_snapshots.go

```diff
@@ -256,6 +266,10 @@ func PrintSnapshots(stdout io.Writer, list restic.Snapshots, reasons []restic.Ke
 			data.Size = ui.FormatBytes(sn.Summary.TotalBytesProcessed)
 		}
 
+		if sn.Description != "" {
+			data.Description = sn.Description
+		}
```

Comment:
```suggestion
		data.Description = sn.Description
```
The if is redundant.